pandas/tests/series/indexing/test_boolean.py

```diff
@@ -128,3 +128,14 @@ def test_get_set_boolean_different_order(string_series):
     sel = string_series[ordered > 0]
     exp = string_series[string_series > 0]
     tm.assert_series_equal(sel, exp)
+
+
+def test_getitem_boolean_dt64_copies():
+    # GH#36210
+    dti = date_range("2016-01-01", periods=4, tz="US/Pacific")
+    key = np.array([True, True, False, False])
```

Comment:
can you replicate the OP here as well (e.g. add the slice test for this too)

Reply:
not sure i understand.  this is exactly the example from the OP

Reply:
the OP had 2 example inputs
```
er1 = pd.Series(dti._data)
ser2 = pd.Series(range(4))
```